src/ingest_validation_tools/table-schemas/level-2/lcms.yaml

```diff
@@ -80,7 +85,7 @@ fields:
     description: units of LC column inner diameter (typically microns)
     constraints:
       enum:
-        - um
+        - micron
```

Comment:
Unless one community strongly prefers micron to um, I think we should be consistent?

Reply:
"micron" is the standard, I am discovering. If you keep, then I will have to change & resubmit a number of metadata.tsvs.